### PR TITLE
Init and Logrotate material for handdansguardian parser

### PR DESCRIPTION
**Pull request by kti-richard**
Init script, boot hook script, and logrotate configuration
to support the Dansguardian category output parser.

See:
http://wiki.yona.nu/display/DEV/Perl+script+to+pass+events+from+SmoothWall+to+the+Yona+server

For details on how to deploy

### CODE REVIEW

**Comment by kti-richard** [comment]
Note that the changes to the perl script itself might be breaking changes for the way it is currently run on older infrastructure.  Since we manually deploy this component though, it should not be a problem.

File: scripts/smoothwall/HandleDansGuardianLog.pl [hunk 1]
```diff
@@ -140,6 +140,21 @@ sub handle_records_from_file {
 	close INPUT_FILE;
 }
 
+BEGIN {
+	#Fork
```
Comment (Bert-R) [inline scripts/smoothwall/HandleDansGuardianLog.pl]: Can you add an explanatory comment why the script needs to fork?

File: scripts/smoothwall/HandleDansGuardianLog.pl [hunk 2]
```diff
@@ -140,6 +140,21 @@ sub handle_records_from_file {
...
+	my $pidFile = '/var/run/HandleDansGuardianLog.pid';
+	my $pid = fork;
+	if ($pid) # parent: save PID
+	{
+		open PIDFILE, ">$pidFile" or die "can't open $pidFile: $!\n";
```
Comment (Bert-R) [inline scripts/smoothwall/HandleDansGuardianLog.pl]: When adding the comment, can you also correct the indentation here?

**Comment by Bert-R** [comment]
Looks good.

**Pull request merged**